pyflakes/messages.py

```diff
@@ -265,7 +265,7 @@ class InvalidPrintSyntax(Message):
 
 
 class IsLiteral(Message):
-    message = 'use ==/!= to compare str, bytes, and int literals'
+    message = 'use ==/!= to compare constant literals (str, bytes, int, float, tuple)'
```

Comment:
For syntax errors pyflakes matches the interpreter error messages. Not sure if we should do the same here.

Reply:
should I change it to:

`"is" / "is not" with a literal.  Did you mean "==" / "!="?`

```console
$ python3.8 -c '1 is 1'
<string>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
$ python3.8 -c '1 is not 1'
<string>:1: SyntaxWarning: "is not" with a literal. Did you mean "!="?
```

Reply:
I don't know how important it is to make them match really.